package.json

```diff
@@ -212,6 +212,44 @@
                 "type": "boolean",
                 "description": "If true, will log all communication between VS Code and the adapter"
               },
+              "proxy": {
+                "type": "object",
+                "properties": {
+                  "allowMultipleSessions": {
+                    "enum": [
+                      0,
+                      1
+                    ],
```

Comment:
Seems like this should be a boolean?

Reply:
This has been updated to be a `boolean`.